client/my-sites/plans/jetpack-plans/product-lightbox/style.scss

```
@@ -173,78 +179,60 @@
 	border-radius: 0 8px 8px 0; /* stylelint-disable-line scales/radii */
 }
 
-.product-lightbox__variants-option {
-	margin-top: 1rem;
-	margin-bottom: 2rem;
-	display: flex;
-	flex-direction: row;
+.product-lightbox__variants-options {
+	.form-legend {
+		font-weight: 600;
+		font-size: 0.875rem;
+		margin-bottom: 0.5rem;
+	}
 
-	.multi-checkbox {
+	.form-fieldset {
+		margin-bottom: 0;
+	}
+
+	.multiple-choice-answers-wrapper {
+		margin-top: 1rem;
+		margin-bottom: 1.5rem;
 		display: flex;
+		flex-direction: row;
+	}
 
-		.form-label {
-			padding-right: 1.5rem;
-			span {
-				font-weight: 600;
-			}
-		}
+	.form-label {
+		clear: none;
+		cursor: pointer;
+		display: inline-block;
+		position: relative;
+		padding-right: 3rem;
 	}
-}
-.product-lightbox__variants-option-label {
-	display: flex;
-	cursor: pointer;
-	user-select: none;
-	& + .radio {
-		margin-top: 12px;
-	}
-	input {
-		display: none;
-		& + span {
-			display: inline-block;
-			position: relative;
-			padding-left: 1.75rem;
-			padding-right: 3rem;
-			font-weight: 600;
-
-			/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
-			&:before {
-				content: '';
-				display: block;
-				position: absolute;
-				border-radius: 50%; /* stylelint-disable-line declaration-property-unit-allowed-list */
-				width: 1rem;
-				height: 1rem;
-				left: 0;
-				top: 2px;
-				border: 2px solid #afb1b6;
-				background: #ffffff;
-			}
-			/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
-			&:after {
-				content: '';
-				display: block;
-				width: 0.75rem;
-				height: 0.75rem;
-				background: #008710;
-				position: absolute;
-				border-radius: 50%; /* stylelint-disable-line declaration-property-unit-allowed-list */
-				top: 6px;
-				left: 4px;
-				opacity: 0;
-				transform: scale( 0, 0 );
-				transition: all 0.2s cubic-bezier( 0.64, 0.57, 0.67, 1.53 );
-			}
-		}
-		/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
-		&:checked + span:after {
-			opacity: 1;
-			transform: scale( 1, 1 );
-		}
 
-		/* stylelint-disable-next-line selector-pseudo-element-colon-notation */
-		&:checked + span:before {
-			border: 2px solid #008710;
+	.form-label input[type='radio'] + span {
+		font-weight: 600;
+		font-size: 1rem;
+		margin-left: 1.75rem;
```

It's better to use [RTL aware](https://hacks.mozilla.org/2015/10/building-rtl-aware-web-apps-websites-part-2/) alternatives:

```suggestion
		margin-inline-start: 1.75rem;
```